main.cpp

```diff
@@ -131,287 +172,412 @@ struct CarWash
+
     void crack();
-//     2) buzz
     void buzz();
-//     3) break a wire
-    bool breakSnareWires(); // returns true to alert SnareDrum struct that something is broken
+    bool breakSnareWires();
 };
 
-// Thing 7) Snare Strainer
+void SnareWires::crack()
+{
+    std::cout<<"Crack!"<<std::endl;
+}
+
+void SnareWires::buzz()
+{
+    std::cout<<"Bzzzzzzz..."<<std::endl;
+}
+
+bool SnareWires::breakSnareWires()
+{
+    std::cout<<"Snare wires have broken!"<<std::endl;
+    return 1;
+}
+
 struct SnareStrainer
 {
-// 5 properties:
-//     1) mounting distance, in inches (float)
     float mountingDistance = 2.0f;
-//     2) number of mounting screws (int)
     int numMountingScrews = 2;
-//     3) maximum tension, in poundsforce (float)
     float maximumTension = 12.5f;
-//     4) brand (std::string)
     std::string brand = "Ludwig";
-//     5) type (std::string)
     std::string strainerType = "side-action";
-// 3 things it can do:
-//     1) engage snares
-    bool engageSnares(); // returns true to snareWires
-//     2) disengage snares
-    bool disengageSnares(); // returns false to snareWires
-//     3) tighten snares
-    float tightenSnares(SnareWires* snareWires, float tension); // tightens or loosens snareWires by given tension
+
+    bool engageSnares();
+    bool disengageSnares();
+    float tightenSnares(SnareWires* snareWires, float tension);
 };
 
-// Thing 8) Drum Shell
+bool SnareStrainer::engageSnares()
+{
+    std::cout<<"Snares engaged!"<<std::endl;
+    return 1;
+}
+
+bool SnareStrainer::disengageSnares()
+{
+    std::cout<<"Snares disengaged."<<std::endl;
+    return 0;
+}
+
+float SnareStrainer::tightenSnares(SnareWires *snareWires, float tension)
+{
+    std::cout<<snareWires<<"have been tightened by "<<tension<<"."<<std::endl;
+    return tension;
+}
+
+
 struct DrumShell
 {
-// 5 properties:
-//     1) brand (std::string)
     std::string brand = "Pearl";
-//     2) material (std::string)
     std::string material = "birch";
-//     3) diameter, in inches (int)
     int diameter = 14;
-//     4) bearing edge angle (float)
     float bearingEdgeAngle = 45.0f;
-//     5) depth, in inches (float)
     float depth = 5.5f;
-// 3 things it can do:
-//     1) crack
-    bool crackShell(); // returns true to alert SnareDrum struct that something is broken
-//     2) be vented
+
+    bool crackShell();
     void beVented(int numHoles, float holeDiameter);
-//     3) resonate
     void resonate(float velocity);
 };
 
-// Thing 9) Lugs
+bool DrumShell::crackShell()
+{
+    std::cout<<"Drum shell has cracked!"<<std::endl;
+    return 1;
+}
+
+void DrumShell::beVented(int numHoles, float holeDiameter)
+{
+    std::cout<<"Drum shell has been vented with "<<numHoles<<"holes of "<<holeDiameter<<" in. in diameter."<<std::endl;
+}
+
+void DrumShell::resonate(float velocity)
+{
+    if(velocity < 35.0f) std::cout<<"no resonance..."<<std::endl;
+    else std::cout<<"resonance with "<<velocity<<" velocity!"<<std::endl;
```

Comment:
Put the bodies of `if` blocks on their own line to make the code easier to read.
- [ ] revise